vendor/src/github.com/docker/engine-api/types/events/events.go

```diff
@@ -9,8 +9,14 @@ const (
 	VolumeEventType = "volume"
 	// NetworkEventType is the event type that networks generate
 	NetworkEventType = "network"
-	// DaemonEventType is the event type that daemon generate
+	// DaemonEventType is the event type that daemon generates
 	DaemonEventType = "daemon"
+	// SwarmEventType is the event type that swarm generates
+	SwarmEventType = "swarm"
```

Comment:
@vdemeester 
While this change is in the vendor of docker, And I have not found PRs in docker/engine-api repo.
May I have your thought on how to plan the procedure?